app/controllers/FeedbackController.scala

```diff
@@ -80,7 +82,8 @@ import scala.concurrent.{ExecutionContext, Future}
   def submit(
     service: Option[String] = None,
     backUrl: Option[String] = None,
-    canOmitComments: Boolean = false
+    canOmitComments: Boolean = false,
+    referrerUrl: Option[String] = None
```

Comment:
@JoPintoPaul do we need it added to this action? Seems like by this point it's being grabbed from the request via `FeedbackFormBind.form.bindFromRequest()`

Reply:
thinking about it, even if it's not used, I guess we want it in the url. But do we have to define every query param that is allowed in the routes or get a routing error? Probably makes sense to leave it just to show that it should be expected to be included even if it wouldn't cause an error to remove it but still supply it